### PR TITLE
Make Cookie-Banner accessible

### PR DESCRIPTION
**Pull request by natalka**
UPDATE LOGO: https://github.com/infopark-customers/bima-immobilien-portal/pull/189

Screenshot 2023-06-30 at 09 05 39
Screenshot 2023-06-30 at 09 05 27

### CODE REVIEW

File: src/Components/Footer.js [hunk 1]
```diff
@@ -33,15 +36,27 @@ export default function Footer() {
...
   return (
     <Modal.Footer>
-      <a role="button" onClick={() => switchBannerMode()} href="#bottom">
+      <button
+        onClick={switchBannerMode}
+        onKeyDown={(keyEvent) =>
+          onKeyAccess(keyEvent, () => switchBannerMode())
+        }
+        className="btn-cta btn btn-no-border"
+      >
```
Comment (natalka) [inline src/Components/Footer.js:45]: We will need to adjust is within other apps using this package 🥲

File: src/Components/CookieGatekeeper.js [hunk 2]
```diff
@@ -23,7 +24,8 @@ function CookieGatekeeper({ children, cookieName }) {
       />
     );
   }
-  return children;
+
+  return <>{children}</>;
```
Comment (jdahlke) [inline src/Components/CookieGatekeeper.js:28]: Why?

File: src/Components/Body.js [hunk 3]
```diff
@@ -42,6 +42,7 @@ export default function Body() {
               title={item.title}
               target="_blank"
               rel="noreferrer"
+              onKeyDown={onLinkAccess}
```
Comment (jdahlke) [inline src/Components/Body.js:45]: Why do we need this? 
Wouldn't it be better to stick to the default behaviour of links?

File: src/Components/CookieBanner.js [hunk 4]
```diff
@@ -18,6 +18,8 @@ export default function CookieBanner() {
       show={bannerVisibility}
       onHide={() => switchBannerMode()}
       scrollable
+      aria-label="cookie-banner"
```
Comment (jdahlke) [inline src/Components/CookieBanner.js]:
```
aria-label={I18n.t(`CookieBanner.label`, { ns: "cookieBanner" })}
```

Translation `de`
```
"Diese Webseite verwendet Cookies"
```

Translation `de`
```
"This website uses cookies"
```

File: src/Components/CookieTypeBlock.js [hunk 5]
```diff
@@ -3,55 +3,60 @@ import Accordion from "react-bootstrap/Accordion";
 import className from "classnames";
 import SingleCookieBlock from "./SingleCookieBlock";
 import { useCookieConsent } from "./CookieConsentContext";
+import { onKeyAccess } from "../utils/accessibilityHelper";
 
 function CookieTypeBlock({ cookieDescription, eventKey }) {
   const { switchCookiesOfType, isCookieTypeAccepted, I18n } =
```
Comment (jdahlke) [inline src/Components/CookieTypeBlock.js:9]: Translation coming from context?!? 🤔

**Review by jdahlke — CHANGES_REQUESTED** [review]

File: src/Components/CookieGatekeeper.js (hunk 2 above)
Comment (natalka) [reply src/Components/CookieGatekeeper.js:28]: Screenshot 2023-07-03 at 09 15 30

File: src/Components/Body.js (hunk 3 above)
Comment (natalka) [reply src/Components/Body.js:45]: it doesn't hurt to handle 'space' for link, but not necessary. can be removed can be kept.

File: src/Components/CookieBanner.js (hunk 4 above)
Comment (natalka) [reply src/Components/CookieBanner.js]: `en.json` and `de.json` are identical files. Probably used at the begging as a default with German content, dunno. I would remove it - what do you think?

File: src/Components/CookieTypeBlock.js (hunk 5 above)
Comment (natalka) [reply src/Components/CookieTypeBlock.js:9]: yes 🤷‍♀️

File: src/Components/CookieBanner.js (hunk 4 above)
Comment (jdahlke) [reply src/Components/CookieBanner.js]: Nope, please provide English translation in a separate pull request. Or I can do it, if you think that's better.

Comment (natalka) [reply src/Components/CookieBanner.js]: https://github.com/infopark-customers/scrivito-another-cookie-banner/pull/2

**Review by jdahlke — APPROVED** [review]

**Pull request merged**